src/Advection/upwind_biased_first_order.jl

```diff
@@ -0,0 +1,33 @@
+#####
+##### Upwind-biased 1rd-order advection scheme
+#####
+
+struct UpwindBiasedFirstOrder <: AbstractUpwindBiasedAdvectionScheme{1} end
+
+const U1 = UpwindBiasedFirstOrder
+
+@inline boundary_buffer(::U1) = 0
+
+@inline symmetric_interpolate_xᶠᵃᵃ(i, j, k, grid, ::U1, c) = 0 
+@inline symmetric_interpolate_yᵃᶠᵃ(i, j, k, grid, ::U1, c) = 0 
+@inline symmetric_interpolate_zᵃᵃᶠ(i, j, k, grid, ::U1, c) = 0 
+
+@inline symmetric_interpolate_xᶜᵃᵃ(i, j, k, grid, ::U1, u) = 0
+@inline symmetric_interpolate_yᵃᶜᵃ(i, j, k, grid, ::U1, v) = 0
+@inline symmetric_interpolate_zᵃᵃᶜ(i, j, k, grid, ::U1, w) = 0
```

Comment:
Is this right? This will mean that we cannot use `UpwindBiasedFirstOrder` for momentum advection. Probably we have to use second-order interpolates here.

Reply:
I admit that I'm a little confused as to how the symmetric interpolate is used.  If we need second-order centered scheme that is easy enough to do.

Reply:
Symmetric interpolation is used to reconstruct the _advecting_ momentum components for the purpose of upwind-biased advection of momentum. For example, the advection of x-momentum by "v" is

https://github.com/CliMA/Oceananigans.jl/blob/77e2d1c8b6418db7ac7b4f4d6de0e362107c0942/src/Advection/upwind_biased_advective_fluxes.jl#L27-L34

Reply:
Thanks that helps.  The code has been updated so I presume this particular concern is resolved.